lib/tab-bar-item-ios.js

```diff
@@ -16,18 +16,22 @@ export default function createTabBarItemIOSComponent(IconNamePropType, getImageS
       iconName: IconNamePropType.isRequired,
       selectedIconName: IconNamePropType,
       iconSize: PropTypes.number,
+      iconColor: PropTypes.string,
+      selectedIconColor: PropTypes.string,
     };
 
     static defaultProps = {
       iconSize: 30,
+      iconColor: '#929292',
+      selectedIconColor: '#929292',
```

Comment:
Hmm, do we really need these defaults? And I'm curious why you chose these defaults :-)